Evaluacion

GCarboni96 opened the pull request.

GCarboni96 committed:
agregamos boton de termina prueba a planilla

GCarboni96 committed:
cambios menores

GCarboni96 committed:
cambios menores

GCarboni96 committed:
cambios menores

barbaracasstro committed:
MUestra modal de feedback para la plantilla de compresion

barbaracasstro committed:
Merge branch 'develop' into evaluacion

barbaracasstro committed:
WIP: crea clase PlantillaCOmpresion para manejar feeback

barbaracasstro committed:
agrega feedback plantilla prueba compresion

GCarboni96 committed:
feedback a otras pruebas añadido, junto a ciertos retoques

GCarboni96 committed:
arreglados ultimos bugs de los feedbacks

GCarboni96 committed:
agregado archivo platillas.js

felipe-canales commented:
No directamente relacionado a este pull request, pero la altura está bien y el feedback me dice que esta mal
![image](https://user-images.githubusercontent.com/22508602/102258315-92247980-3eec-11eb-9ceb-428ddcc285a2.png)
![image](https://user-images.githubusercontent.com/22508602/102258351-9cdf0e80-3eec-11eb-8edf-199e60cecddb.png)

felipe-canales commented:
Al presionar "Terminar prueba" en la prueba de rendimiento, no es obvio que hay que scrollear hacia arriba.
Tal vez esto ayudaría: https://stackoverflow.com/questions/14411176/window-location-with-the-same-pagediv-id
![image](https://user-images.githubusercontent.com/22508602/102259500-f267eb00-3eed-11eb-9617-ad250d3a9214.png)

GCarboni96 committed:
arreglada vista de terminar prueba, ahora el scroll para ver los resultados del feedback es automatico

GCarboni96 committed:
Merge branch 'develop' into evaluacion

GCarboni96 committed:
mergeado con rama develop y modificado main.js para que conserve el diccionario de errores

barbaracasstro committed:
Termina feedback prueba imagen

barbaracasstro committed:
Boton "FInalizar prueba" redirige al inicio de simulador

GCarboni96 committed:
Merge branch 'develop' into evaluacion

barbaracasstro committed:
Orden en tablas plantilla rendimiento

barbaracasstro committed:
Merge branch 'evaluacion' of https://github.com/cc5402-Mamografias/mamografsim into evaluacion

barbaracasstro committed:
Borra código sin funcionalidad plantilla imagen

barbaracasstro committed:
Borra código sin funcionalidad plantilla rendi

barbaracasstro committed:
Borra espacios

barbaracasstro committed:
Se agrega clase titulo-seccion a pasos

barbaracasstro committed:
Borra código extra (divs)

barbaracasstro committed:
Agrega estilo Firefox

Se agrega opcion para que tabla quede centrada en el navegador firefox, en la plantilla de rendimiento

barbaracasstro committed:
Agrega estilo para dejar bullets centrados
